style.css

```diff
@@ -0,0 +1,343 @@
+
+/* google font  */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #396df2;
+  --clr-gray-100: #808080;
+}
+
+/* Reset style for page */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* navbar styles */
+.header-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem;
+}
+
+.topNav {
+  display: none;
+}
+
+.nav-items {
+  margin: 0 0.5rem;
+}
+
+.nav-brand {
+  font-weight: 700;
+  color: #6070ff;
+}
+
+/* Hamburger menu */
+.hamburger {
+  display: block;
+}
+
+.bar {
+  display: block;
+  height: 5px;
+  width: 35px;
+  background-color: #6070ff;
+  margin: 3px;
+}
+
+/* background shadow  */
+.box2 {
+  display: flex;
+  flex-direction: column;
+  box-shadow: -10px 10px 10px -6px rgba(207, 205, 205, 0.501);
+  padding-bottom: 0;
+  border-bottom-left-radius: 3rem;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.Greetings {
+  margin: 0 0;
+}
+
+.Greetings h1 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 4rem;
```

Comment:
:red_circle: [REQUIRED] Please make sure you reduce the text size of this header to match the Figma design. On the Figma Design, this value is set to 40px. the 4rem value after  being computed is  60px

Your Design
![header](https://user-images.githubusercontent.com/26736582/205330903-24a3e84e-3ac3-45de-8360-b3d304552ee9.png)

Figma Design
![figma header](https://user-images.githubusercontent.com/26736582/205330998-03357c97-ade5-498f-80d6-4e49828661ba.png)

Reply:
- [ ] 🛑  as mentioned by the previous reviewer , kindly reduce the size of your header's title